django/src/rdwatch/server/settings.py

```diff
@@ -32,8 +32,34 @@ class BaseConfiguration(Configuration):
         'django_extensions',
         'rdwatch',
         'django_celery_results',
+        # Maybe conditionally add Scoring
+        'rdwatch_scoring',
     ]
 
+    @property
+    def DATABASES(self):
+        DB_val = values.DatabaseURLValue(
+            alias='default',
+            environ_name='POSTGRESQL_URI',
+            environ_prefix=_environ_prefix,
+            environ_required=True,
+            # Additional kwargs to DatabaseURLValue are passed to dj-database-url
+            engine='django.contrib.gis.db.backends.postgis',
+        )
+        db_dict = DB_val.value
+        if os.environ['RDWATCH_POSTGRESQL_SCORING_URI'] is not None:
```

Comment:
This will crash with a `KeyError` if `'RDWATCH_POSTGRESQL_SCORING_URI'` is not in os.environ.
```suggestion
        if 'RDWATCH_POSTGRESQL_SCORING_URI' in os.environ:
```